backends/arm/test/ops/test_full.py

```diff
@@ -0,0 +1,159 @@
+                module,
+                example_inputs=test_data,
+                compile_spec=common.get_tosa_compile_spec(
+                    permute_memory_to_nhwc=permute_memory_to_nhwc
+                ),
+            )
+            .quantize()
+            .export()
+            .check_count({"torch.ops.aten.full.default": 1})
+            .to_edge()
+            .partition()
+            .check_not(["executorch_exir_dialects_edge__ops_aten_full_default"])
+            .check_count({"torch.ops.higher_order.executorch_call_delegate": 1})
+            .to_executorch()
+            .run_method_and_compare_outputs(inputs=test_data)
+        )
+
+    def _test_full_tosa_u55_pipeline(
+        self, module: torch.nn.Module, test_data: Tuple[torch.Tensor]
+    ):
+        (
+            ArmTester(
+                module,
+                example_inputs=test_data,
+                compile_spec=common.get_u55_compile_spec(),
+            )
+            .quantize()
+            .export()
+            .check_count({"torch.ops.aten.full.default": 1})
+            .to_edge()
+            .partition()
+            .check_not(["executorch_exir_dialects_edge__ops_aten_full_default"])
+            .check_count({"torch.ops.higher_order.executorch_call_delegate": 1})
+            .to_executorch()
+        )
+
+    def test_only_full_tosa_MI(self):
+        self._test_full_tosa_MI_pipeline(self.Full(), ())
+
+    def test_const_full_tosa_MI(self):
+        _input = torch.rand((2, 2, 3, 3)) * 10
+        self._test_full_tosa_MI_pipeline(self.AddConstFull(), (_input,))
+
+    def test_const_full_nhwc_tosa_BI(self):
+        _input = torch.rand((2, 2, 3, 3)) * 10
+        self._test_full_tosa_BI_pipeline(self.AddConstFull(), (_input,), True)
+
+    @parameterized.expand(AddVariableFull.test_parameters)
+    def test_full_tosa_MI(self, test_tensor: torch.Tensor):
+        self._test_full_tosa_MI_pipeline(self.AddVariableFull(), test_tensor)
+
+    @parameterized.expand(AddVariableFull.test_parameters)
+    def test_full_tosa_BI(self, test_tensor: torch.Tensor):
+        self._test_full_tosa_BI_pipeline(self.AddVariableFull(), test_tensor, False)
+
+    @parameterized.expand(AddVariableFull.test_parameters)
+    def test_full_u55_BI(self, test_tensor: torch.Tensor):
+        self._test_full_tosa_u55_pipeline(
+            self.AddVariableFull(),
+            test_tensor,
+        )
+
+    # This fails since full outputs int64 by default if 'fill_value' is integer, which our backend doesn't support.
+    @unittest.expectedFailure
+    def test_integer_value(self):
+        _input = torch.ones((2, 2))
+        integer_fill_value = 1
+        self._test_full_tosa_MI_pipeline(
+            self.AddVariableFull(), example_data=(_input, integer_fill_value)
+        )
+
+    # This fails since the fill value in the full tensor is set at compile time by the example data (4).
+    # Test data tries to set it again at runtime (to 5) but it doesn't do anything.
```

Comment:
This is tricky I think, how do you detect this i.e. partition only if the value won't change at runtime? But I think PT compiler also specialized so this might be OK.

Reply:
From what I found, this was a limitation in the export call, the shape/ fill value parameters became an attributes rather than tensors in the fx graph (but please correct me if I'm wrong).

Reply:
Yeah I think so too.